js/src/Ice/Stream.js

```diff
@@ -2299,14 +2299,13 @@ export class OutputStream {
                 instance = arg1;
             } else if (arg1.constructor == EncodingVersion) {
                 this._encoding = arg1;
-            } else {
-                throw new InitializationException("unknown argument to OutputStream constructor");
-            }
-        }
-
-        if (arg2 !== undefined && arg2 !== null) {
-            if (arg2.constructor == EncodingVersion) {
-                this._encoding = arg2;
+                if (arg2 !== null && arg2 !== undefined) {
```

Comment:
Should we remove the Instance flavor above?

If you have an instance, it's trivial to use the (encoding, format) constructor.